bin/i18n/resources/dashboard/marketing_announcements/sync_in.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/sync_in_base'
+require_relative '../marketing_announcements'
+
+module I18n
+  module Resources
+    module Dashboard
+      module MarketingAnnouncements
+        class SyncIn < I18n::Utils::SyncInBase
+          def process
+            prepare_marketing_announcements_data
+          end
+
+          private
+
+          def prepare_marketing_announcements_data
+            banners = I18nScriptUtils.parse_file(ORIGIN_I18N_FILE_PATH)['banners']
+            banners.each_value do |banner|
+              banner.slice!('title', 'body', 'buttonText')
+            end
+            I18nScriptUtils.write_json_file(I18N_SOURCE_FILE_PATH, {'banners' => banners})
```

Comment:
https://github.com/rubocop/ruby-style-guide#empty-lines-between-methods
```suggestion
            banners = I18nScriptUtils.parse_file(ORIGIN_I18N_FILE_PATH)['banners']

            banners.each_value do |banner|
              banner.slice!('title', 'body', 'buttonText')
            end

            I18nScriptUtils.write_json_file(I18N_SOURCE_FILE_PATH, {'banners' => banners})
```